src/examples/crypto/ecdsa/ecdsa.ts

```diff
@@ -27,14 +27,6 @@ const keccakAndEcdsa = ZkProgram({
         return signature.verify(message, publicKey);
       },
     },
-
-    sha3: {
-      privateInputs: [],
-      method(message: Bytes32) {
```

Comment:
I felt this was clobbering up the ecdsa test, and is covered in another circuit regression test already